molo/commenting/models.py

```diff
@@ -19,3 +21,9 @@ class MPTTMeta:
     class Meta:
         app_label = 'commenting'
         ordering = ['-submit_date', 'tree_id', 'lft']
+        
+@receiver(comment_was_flagged, sender=MoloComment)
+def remove_comment_if_flag_limit(sender, comment, flag, created, **kwargs):
+    if (comment.flags.count() >= 3):
```

Comment:
You can just have this as `COMMENTS_FLAG_THRESHHOLD = 3` in `settings/base.py` and just use 
```python
from django.conf import settings
...
if (comment.flags.count() >= settings.COMMENTS_FLAG_THRESHHOLD)
```